frontend/src/layout/header/UserBox.tsx

```diff
@@ -282,15 +282,28 @@ const FloatingMenu: React.FC<FloatingMenuProps> = ({ close, type }) => {
     );
 };
 
+// Use this for text elements that exist exclusively for screen reader purposes
+export const visuallyHidden = {
+    clipPath: "inset(50%)",
+    height: 1,
+    width: 1,
+    overflow: "hidden",
+    position: "absolute",
+    whiteSpace: "nowrap",
+} as const;
+
 
 export const LanguageSettings: React.FC = () => {
     const { t } = useTranslation();
 
-    return <WithFloatingMenu type="language">
-        <ActionIcon title={t("language")}>
-            <HiOutlineTranslate />
-        </ActionIcon>
-    </WithFloatingMenu>;
+    return <>
+        <div css={visuallyHidden} aria-live="polite">{t("selected-language")}</div>
```

Comment:
So this thing... I'm not sure. The original issue this PR fixes doesn't mention the need for this. And I'm not sure either. The checkbox things totally make sense to me (what the original issues talks about). But is this really something good for accessibility? :thinking: It's the first thing of its kind we add and I'm not sure this announcment is necessary? Can you explain your reasoning for adding this?

Reply:
The original issue includes the following:
> This should also fix the second problem that right now, users get no feedback at all when actually selecting another language. So you don't know if your choice was accepted before you start tabbing though other interface elements again.

That's why I added this. Maybe I misunderstood something in there?

Reply:
Mh no I think you understood Lars correctly, I'm just not sure if I agree with him :D 
I mean, I don't know, I am not a screen reader user. But I am thinking: the last read text is "checkbox english, not checked". And then the user presses enter and presumably something new is read because the focus shifts. But even if not, the focus is still in the header, so any tab or shift+tab will lead you to an element with a "translated label". So to me it seems like it's sufficiently clear what happened. Ok sure, just pressing enter... there might be an error or something. But just tabbing once to see if stuff is now in the correct language seems like an easy thing to do. And it's not like people will likely change their language all the time.

So I would think it's not all too useful and we can rather avoid having this weird hidden div. But... I think we should ask someone who is actually a screen reader user...

Reply:
Could you create another draft PR without this hidden div? Then we can send out two test deployment links to people to let them test. That would be great!